src/main/java/com/lunar/world/WorldMap.java

```diff
@@ -70,7 +70,7 @@ private void setMultiple(Tile tile, int startX, int startY, Direction direction,
         int width = tile.getWidth();
         int height = tile.getHeight();
 
-        while (iterations <= amount) {
```

Comment:
Is there any reason to not use a fori loop?

```java
for (int i = 0; i < amount; i++) {
    // Code
}
```

Reply:
Agreed, it is strange just left it as I found it.